### PR TITLE
Don't generate parameters.txt if no GEN_KW is specified [ERT-1337]

### PR DESCRIPTION
**Pull request by andreabrambilla**
**Task**
Avoid creating "parameters.txt" file when no GEN_KW is specified (ERT-1337)

**Approach**
Check the number of config nodes of var_type PARAMETER before creating the file

**Pre un-WIP checklist**
- [x] Statoil tests pass locally
- [x] Have completed graphical integration test steps

**Depends on**
* Statoil/libecl#

### CODE REVIEW

**Comment by andreabrambilla** [comment]
NOTE: I'm planning to add a test for this in /private/abramb/workspace/statoil/libres/python/tests/res/enkf/test_enkf.py but I'm having problems with my IDE at the moment, I'll do it on Monday

File: libenkf/src/enkf_state.c [hunk 1]
```diff
@@ -1223,40 +1223,40 @@ void enkf_state_ecl_write(enkf_state_type * enkf_state, const run_arg_type * run
...
 
-  const shared_info_type * shared_info   = enkf_state->shared_info;
-  const model_config_type * model_config = shared_info->model_config;
-  int iens                               = enkf_state_get_iens( enkf_state );
-  const char * base_name                 = model_config_get_gen_kw_export_file(model_config);
-  char * export_file_name                = util_alloc_filename( run_arg_get_runpath( run_arg ) , base_name  , NULL);
-  FILE * export_file                     = util_mkdir_fopen(export_file_name, "w");
-
-
   stringlist_type * key_list = ensemble_config_alloc_keylist_from_var_type( enkf_state->ensemble_config , PARAMETER );
```
Comment (joakim-hove) [inline libenkf/src/enkf_state.c:1227]: This should be changed to: 
```C
stringlist_type * key_list = ensemble_config_alloc_keylist_from_impl_type( enkf_state->ensemble_config , GEN_KW);
```

**Comment by joakim-hove** [comment]
Ping?

**Comment by andreabrambilla** [comment]
Failure seems to be related to an issue on Travis, jenkins job passed with no problems

**Pull request merged**